### ISSUE TITLE
node-inspector crashes if there's console.log in a callback function

### ISSUE THREAD

**Issue by sinkingshriek** [node-inspector/node-inspector#517]
This was first discovered in here https://github.com/nodeschool/austin/issues/4#.

Reproduced on:-
- Node v0.11.13
- Node Inspector v0.8.1
- Chrome 39.02(64bit)
- Ubuntu 14.04
- More env tested on the [above link](https://github.com/nodeschool/austin/issues/4#issuecomment-65733435) by @dcbartlett 

From my first impression I initially thought this had to do with `lodash` not playing well with `node-inspector` but upon my further investigation I could reproduce this issue on simple `forEach` as well. e.g.

    ["hello","world"].forEach(function(i){
        //crashes on this console
	    console.log('inside a callback');
    });
    console.log("never gets here");

So basically, node-inspector crashes whenever it sees `console` in a callback.
I also did the following to keep the code simple with no logics and loops.

    var obj = {
	    someFunc: function(callback){
            //works fine here
    		console.log("Inside normal function");
    		callback();
    	}
    };
    obj.someFunc(function(){
        //dies here
	    console.log("inside callback function");
    });

At the moment using `util.log` is an alternative solution as discussed in the original issue tracker but would love for this to be resolved too.

**Comment by 3y3** [comment]
@sinkingshriek , I commited fixup just now. You can fork master HEAD or wait for releasing 0.8.2

**Comment by sinkingshriek** [comment]
Wow, that was quick. :laughing: 
Tested with your recent check-in. Works like a charm.
Thanks for the quick fix. :+1:
